modules/comments/components/AccountComments.tsx

```diff
@@ -58,10 +58,10 @@ export default function AccountComments({ address }: { address: string }): React
           >
             Comments
           </Text>{' '}
-          {data.comments.map(comment => (
+          {data.comments.map((comment, i) => (
             <Box
               sx={{ borderBottom: '1px solid', borderColor: 'secondaryMuted', py: 4 }}
-              key={comment.address.address}
+              key={comment.address.address + '-' + i}
```

Comment:
Do we have anything else unique on the comment object we can use as a key? Maybe the date? Using a mapping index as a key can lead to weird behavior in some situations (probably fine in this case since we're not adding/removing things, but still would be better to avoid it if possible).